### DISCUSSION TITLE
currently selected sort order is not highlighted

### DISCUSSION THREAD

**Discussion by Oplopanax-horridus** [Docile-Alligator/Infinity-For-Reddit#993] [Completed Ideas] [answered]
When looking at sort options for posts or comments, the currently selected option should be highlighted. This should also include the time frame for those options that include that qualifier.

**Comment by Docile-Alligator** [comment]
Good idea.

**Comment by Docile-Alligator** [comment, marked as answer]
Done. [Show an indicator for the current sort type.](https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/965d8ab8aec779d9a300434dd20840f481e3941c)
